lib/safer_rails_console.rb

```diff
@@ -51,7 +51,9 @@ class Configuration
     }.freeze
 
     CONFIG_DEFAULTS.each do |name, value|
-      config_accessor(name) { value }
+      config_accessor(name) do
+        ENV.fetch("SAFER_RAILS_CONSOLE_#{name.upcase}", value)
```

Comment:
It doesn't look like this override mechanism will work for non-string settings. Should we either only support it for string settings or extend it to support booleans, arrays of strings, and hashes?